internal/api/v2/README.md

```diff
@@ -14,4 +14,17 @@ or a specified number of requests, and then dispatch all the requests in one
 batch for each remote API. For example, if the queue duration is 1 second and
 the queue depth is 100, requests will be dispatched 1 second after the request
 that started the queue, or after the queue reaches 100 requests, which ever
-comes first.
\ No newline at end of file
+comes first.
+
+## Migrating from v1
+
+* Query methods are now prefixed with `query`.
+* `version` and `metrics` are unchanged.
+* Query methods are general - you can query any TX with `query-tx`, etc.
+* There is a general `execute` method that will accept arbitrary
+  already-marshalled transaction blobs.
+* All transaction methods are `{action}-{noun}`, e.g. `create-adi`.
```

Comment:
since I have to make a comment, I do like this definition of how commands are defined.  Expanding this to all current and future commands, can help ensure command consistency.